### PR TITLE
Dont run StopwatchDisposable_MeasuresPerformanceCorrectly test in Mac OSX

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Bitcoin.Tests/Utilities/StopwatchTest.cs [hunk 1]
```diff
@@ -31,6 +32,11 @@ public class StopwatchTest
         [Fact]
         public void StopwatchDisposable_MeasuresPerformanceCorrectly()
         {
+            //Don't run this test in a Mac environment as it takes too long,
```
Comment (Aprogiena) [inline src/Stratis.Bitcoin.Tests/Utilities/StopwatchTest.cs]: space after `//` :D

**Comment by bokobza** [comment]
Do we know why this is happening?

**Comment by fassadlr** [comment]
@Aprogiena can you comment?

**Comment by Aprogiena** [comment]
@bokobza nope :( we don't, the mac is so slow that we expect the virtual machine is somehow messed up, or possibly it is run in parallel with something cpu heavy so it does not really move, the measured values are so unreal that for sure there is something wrong.

**Pull request merged**